Test/CoreSDK.Test/Operation.AL.Shared.Tests/TelemetryClientExtensionTests.cs

```diff
@@ -184,5 +184,78 @@ public void StartDependencyTrackingStoresTheArgumentOperationNameInContext()
             var operation = this.telemetryClient.StartOperation<DependencyTelemetry>("TestOperationName");
             Assert.AreEqual("TestOperationName", AsyncLocalHelpers.GetCurrentOperationContext().RootOperationName);
         }
+
+        [TestMethod]
+        public void ContextPropagatesThroughNestedOperations()
+        {
+            using (var operation1 = this.telemetryClient.StartOperation<RequestTelemetry>("OuterRequest"))
+            {
+                using (var operation2 = this.telemetryClient.StartOperation<DependencyTelemetry>("DependentCall"))
```

Comment:
nitpick: I have a strong personal dislike of using var instead of known types but in these usings in particular it really isn't clear what the type returned by StartOperation is.